js/src/functions/invoke.ts

```diff
@@ -187,3 +187,57 @@ export async function invoke<Input, Output, Stream extends boolean = false>(
     return (schema ? schema.parse(data) : data) as InvokeReturn<Stream, Output>;
   }
 }
+
+/**
+ * Creates a function that can be used as a task or scorer in the Braintrust evaluation framework.
+ * The returned function wraps a Braintrust function and can be passed directly to Eval().
+ *
+ * When used as a task:
+ * ```ts
+ * const myFunction = useFunction({projectName: "myproject", slug: "myfunction"});
+ * await Eval("test", {
+ *   task: myFunction,
+ *   data: testData,
+ *   scores: [...]
+ * });
+ * ```
+ *
+ * When used as a scorer:
+ * ```ts
+ * const myScorer = useFunction({projectName: "myproject", slug: "myscorer"});
+ * await Eval("test", {
+ *   task: someTask,
+ *   data: testData,
+ *   scores: [myScorer]
+ * });
+ * ```
+ *
+ * @param options Options for the function.
+ * @param options.projectName The project name containing the function.
+ * @param options.slug The slug of the function to invoke.
+ * @param options.version Optional version of the function to use. Defaults to latest.
+ * @returns A function that can be used as a task or scorer in Eval().
+ */
+export function useFunction({
```

Comment:
`use` is a very special prefix in JS because it implies a hook in React. Let's think about other naming options offline. Also may be good to be consistent with the framework2 style (e.g. `project.scorer.create`, `project.scorer.load`, etc)?

Reply:
Done